cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py

```diff
@@ -114,22 +155,27 @@ def test_migrate_keystone_users(self):
         dst_users = self.dst_cloud.keystoneclient.users.list()
 
         for param in ['name', 'email']:
-            self.validate_resource_parameter_in_dst(src_users, dst_users,
-                                                    resource_name='user',
-                                                    parameter=param)
+            self.validate_tenant_resource_parameter_in_dst(src_users,
+                                                           dst_users,
+                                                           'user',
```

Comment:
please return 'resource_name' and 'parameter' specifying, it makes code more understandable